render/egl.c

```diff
@@ -287,26 +287,12 @@ static bool egl_init(struct wlr_egl *egl, EGLenum platform,
 			check_egl_ext(device_exts_str, "EGL_EXT_device_drm");
 	}
 
-	if (config_attribs != NULL) {
-		EGLint matched = 0;
-		if (!eglChooseConfig(egl->display, config_attribs, &egl->config, 1,
-				&matched)) {
-			wlr_log(WLR_ERROR, "eglChooseConfig failed");
-			return false;
-		}
-		if (matched == 0) {
-			wlr_log(WLR_ERROR, "Failed to match an EGL config");
-			return false;
-		}
-	} else {
-		if (!check_egl_ext(display_exts_str, "EGL_KHR_no_config_context") &&
-				!check_egl_ext(display_exts_str, "EGL_MESA_configless_context")) {
-			wlr_log(WLR_ERROR, "EGL_KHR_no_config_context or "
-				"EGL_MESA_configless_context not supported");
-			return false;
-		}
-		egl->config = EGL_NO_CONFIG_KHR;
-	}
+      	if (!check_egl_ext(display_exts_str, "EGL_KHR_no_config_context") &&
```

Comment:
The indentation looks broken here.

Reply:
Yikes... Oh dear. I had an inkling that *something* was going on, but I thought it was the comparison tool was being a bit funny. Maybe the way my text editor does outdenting probably isn't so good after all. I'll try and see if I can deal with the weird indentation issue manually. Thanks for pointing it out!

Reply:
Never mind, I think it's just my text editor outdenting whatever was already there and I don't think it's doing any funny things behind my back.

By the way... Regardless, I think I fixed it now. I just don't know how to squash commits and on GitHub, unless you're a maintainer, you can't squash commits, AFAIK. I'm not sure if @emersion will do it or if I have to do it myself, but I'll wait and see what happens...

Reply:
You can squash locally and do a force push to the branch, for squashing see https://git-rebase.io/.

Reply:
There, I think that does it! Now to move onto the examples, like @emersion said....

Reply:
It looks like I've fixed everything(including the indentation), so I don't think this is an issue anymore.

And before the latest commits got reverted and while I was working on the PR earlier, it looked like the indentation issue that @Emantor pointed out and I fixed(which was that multiple spaces were used instead of a single tab) was probably caused by something on @bl4ckb0ne's end. That's not to shame, drag or rag on them, it's just something that I think I noticed. Just an FYI thing, that's all.